proto/heimdall/staking/v1beta1/query.proto

```diff
@@ -16,20 +16,27 @@ service Query {
     // Validator queries the validator that match by validator id.
     rpc Validator(QueryValidatorRequest) returns (QueryValidatorResponse) {
         option (google.api.http).get =
-            "/heimdall/base/v1beta1/validator/{validator_id}";
+            "/heimdall/staking/v1beta1/validator/{validator_id}";
     }
 
     // ValidatorSet queries the Current Validator Set
     rpc ValidatorSet(QueryValidatorSetRequest)
         returns (QueryValidatorSetResponse) {
-        option (google.api.http).get = "/heimdall/base/v1beta1/validatorSet";
+        option (google.api.http).get =
+            "/heimdall/staking/v1beta1/validator-set";
+    }
+
+    //
+    rpc StakingOldTx(QueryStakingOldTxRequest)
+        returns (QueryStakingOldTxResponse) {
+        option (google.api.http).get = "/heimdall/staking/v1beta1/isoldtx";
     }
 }
 
 // QueryValidatorRequest is request type for the Query/Validator RPC method
 message QueryValidatorRequest {
     // validator_id defines the validator id to query for.
-    heimdall.types.ValidatorID validator_id = 1;
+    int32 validator_id = 1;
```

Comment:
Why can't we use ValidatorID? @saikumar0x1967

Reply:
validator id not working in query params of grpc gateway routes